Makefile

```diff
@@ -2,6 +2,7 @@ lint:
 	golangci-lint run --config golangci.yaml
 
 pre-pr: lint
+	$(MAKE) -C ./example run
```

Comment:
I added run to the pre-pr.  Since, for the moment, it is our only "test", seems like it is worth running as a pre-pr step.  Thoughts?

Reply:
I like this, it's too easy to change something and forget to verify it actually works if the example wasm is not rebuilt.